devtools/common/src/main/filtered/extensions.json

```diff
@@ -230,6 +230,14 @@
     "artifactId": "quarkus-kotlin",
     "guide": "https://quarkus.io/guides/kotlin"
   },
+  {
+    "name": "Scala",
+    "labels": [
+      "scala"
+    ],
+    "groupId": "io.quarkus",
+    "artifactId": "quarkus-scala"
+  },
```

Comment:
Please let's respect the alphabetical order.

Reply:
Fixed it